src/main/Yardarm/YardarmGenerationSettings.cs

```diff
@@ -29,6 +29,8 @@ public class YardarmGenerationSettings
         public Version Version { get; set; } = new Version(1, 0, 0);
         public string? VersionSuffix { get; set; }
         public string Author { get; set; } = "anonymous";
+        public string Owner { get; set; } = "anonymous";
+        public string Description { get; set; }
```

Comment:
This one needs a nullable annotation.

```suggestion
        public string? Description { get; set; }
```